include/openPMD/Mesh.hpp

```diff
@@ -57,7 +57,8 @@ class Mesh : public BaseRecord< MeshRecordComponent >
         cartesian,
         thetaMode,
         cylindrical,
-        spherical
+        spherical,
+        custom
```

Comment:
the naming we use in the openPMD standard is `other` for such kinds

Reply:
https://github.com/openPMD/openPMD-standard/blob/upcoming-2.0.0/STANDARD.md#required-attributes-for-each-mesh-record

We can do this: `other:<userString>`